README.md

```diff
@@ -38,7 +38,7 @@ It is possible to install MyoSuite with:
 ``` bash
 pip install -U myosuite
 ```
-for advanced installation, see [here](setup/README.md).
+for advanced installation, see [here](https://myosuite.readthedocs.io/en/latest/install.html#alternative-installing-from-source).
```

Comment:
from here -- https://myosuite.readthedocs.io/en/latest/install.html#alternative-installing-from-source

1. `pip install -e -r requirements` ==> `pip install -e .`
2. Remove the options `export PYTHONPATH="<path/to/myosuite>:$PYTHONPATH"`. This pollutes the whole path for all the envs and is a recipe for complications.

Reply:
@Vittorio-Caggiano  -- what about this?